src/complex/Core/Application.hpp

```diff
@@ -118,6 +118,18 @@ class COMPLEX_EXPORT Application
    */
   std::filesystem::path getCurrentDir() const;
 
+  /**
+   * @brief Returns the Complex filter UUID [v4] from the SIMPL filter UUID [v5]
+   * @return std::optional<Uuid>
+   */
+  std::optional<Uuid> getComplexUuid(const Uuid& simplUuid);
+
+  /**
+   * @brief Returns the SIMPL filter UUID(s) [v5] from the Complex filter UUID [v4]
+   * @return std::optional<std::vector<Uuid>>
+   */
+  std::optional<std::vector<Uuid>> getSimplUuid(const Uuid& complexUuid);
```

Comment:
I think it's ok to return `std::vector<Uuid>` here because there isn't a special case for an empty vector that needs to be handled. An empty vector will mean that there were no associated SIMPL filters.